modules/cnf-performance-profile-creator-arguments.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+
+
+[id="performance-profile-creator-arguments_{context}"]
+=  Performance Profile Creator arguments
+
+.Performance Profile Creator arguments
+["cols="30%,70%",options="header"]
```

Comment:
It looks like you're trying to make the first comment smaller, which is good, but that's not rendering properly in the build.

You have an extra quote at the beginning (before cols), which I think is throwing it off.